apps/laboratory/tests/wallet.spec.ts

```diff
@@ -78,3 +78,16 @@ testConnectedMW('it should show multiple accounts', async ({ modalPage, modalVal
   await modalValidator.expectMultipleAccounts()
   await modalPage.closeModal()
 })
+
+testConnectedMW(
+  'it should show Switch Network modal if network is not supported',
+  async ({ modalPage, modalValidator, walletPage }) => {
+    if (modalPage.library === 'solana') {
+      return
+    }
```

Comment:
@ganchoradkov now that RPC supports solana testnet/devnet can we add them to react-wallet?